crates/epaint/src/tessellator.rs

```diff
@@ -904,7 +956,7 @@ fn stroke_path(
                 let p = p1.pos;
                 let n = p1.normal;
                 out.colored_vertex(p + n * feathering, color_outer);
-                out.colored_vertex(p, color_inner);
+                out.colored_vertex(p, get_color(color_inner, p));
```

Comment:
This no longer fades out as it gets thinner

Reply:
resolved with 5451e3043796568c4bce0895d01cfd118e2d1f01